Signal/src/ViewControllers/DebugUI/ContactsUtils.h

```diff
@@ -0,0 +1,25 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class CNContact;
+
+@interface ContactsUtils : NSObject
```

Comment:
Maybe DebugContactUtils? TIOLI, but we usually include the term "Debug" in these kinds of classes.

Reply:
Done.